BioMed.Api/Bio.Med.Infrastructure/Class1.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Bio.Med.Infrastructure
+{
+    public class Class1
+    {
+
+    }
+}
```

Comment:
This is why the **Class1** class was created ?

Reply:
I think because without a class it is not possible to open a library